env

```diff
@@ -0,0 +1 @@
+CLUSTER_WORKERS=5
```

Comment:
@pdhanoyasplunk Can you please clarify if this change is intended?

Reply:
Thanks for catching. Removed.